### PR TITLE
Fixup landing page

### PR DESCRIPTION
**Pull request by IamPete1**
This fixes the image links on the landing page and adds a copy of the filter tool logo as "toplogo" which is pointed to by the AP webpage, this allows us to update the logo easily in the future. 

https://github.com/ArduPilot/ardupilot/pull/25465

### CODE REVIEW

**Review by tridge — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h2>Hardware Report</h2>
 
   <tr>
     <td>
-      <a href="./FilterReivew" style="margin: 20px; display: inline-block;"><img src="../images/FilterReview_Icon.png" style="width:200px"></a>
+      <a href="./FilterReivew" style="margin: 20px; display: inline-block;"><img src="images/FilterReview_Icon.png" style="width:200px"></a>
```
Comment (timtuxworth) [inline index.html:50]: Typo - FilterReivew s/be FilterReview - so the link is broken

File: index.html [hunk 2]
```diff
@@ -36,7 +36,7 @@ <h2>UAV Log viewer</h2>
 
   <tr>
     <td>
-      <a href="./HardwareReport" style="margin: 20px; display: inline-block;"><img src="../images/HardwareReport_Icon.png" style="width:200px"></a>
+      <a href="./HardwareReport" style="margin: 20px; display: inline-block;"><img src="images/HardwareReport_Icon.png" style="width:200px"></a>
```
Comment (timtuxworth) [inline index.html:39]: The filetype is PNG (in caps) this server must be linux based I guess, I tested with this name the image is broken, but if I put in PNG it works.

File: index.html (hunk 1 above)
Comment (timtuxworth) [reply index.html:50]: Also the icon image should be PNG not png (as above)

File: index.html [hunk 3]
```diff
@@ -57,7 +57,7 @@ <h2>Filter Review</h2>
 
   <tr>
     <td>
-      <a href="./PIDReivew" style="margin: 20px; display: inline-block;"><img src="../images/PIDReview_Icon.png" style="width:200px"></a>
+      <a href="./PIDReivew" style="margin: 20px; display: inline-block;"><img src="images/PIDReview_Icon.png" style="width:200px"></a>
```
Comment (timtuxworth) [inline index.html:60]: ditto re PNG vs png

File: index.html [hunk 4]
```diff
@@ -68,7 +68,7 @@ <h2>PID Review</h2>
 
   <tr>
     <td>
-      <a href="./FilterTool" style="margin: 20px; display: inline-block;"><img src="../images/FilterTool_Icon.png" style="width:200px"></a>
+      <a href="./FilterTool" style="margin: 20px; display: inline-block;"><img src="images/FilterTool_Icon.png" style="width:200px"></a>
```
Comment (timtuxworth) [inline index.html:71]: I know it was not changed in this PR, but the description has this awkward explanation "For filter setup the FilterReview tool it is recommended it provides the same bode plot feedback." - could you clean that up?

**Review by timtuxworth — COMMENTED** [review]
This is awesome, I didn't know we had this. A couple of cleanups ...

**Pull request merged**